### PR TITLE
Fix hybrid common upgrades automation

### PR DESCRIPTION
**Pull request by tonidero**
We were using single quotes for the podspec file instead of double quotes. That was making the automation to update hybrid common fail. This fixes that and updates to PHC 6.2.0

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ files_with_version_number = {
     './android/src/main/java/com/revenuecat/purchases/capacitor/PurchasesPlugin.kt' => ['PLUGIN_VERSION = "{x}"']
 }
 files_to_update_phc_version = {
-    'RevenueCatPurchasesCapacitor.podspec' => ['"PurchasesHybridCommon", \'{x}\''],
+    'RevenueCatPurchasesCapacitor.podspec' => ["'PurchasesHybridCommon', \'{x}\'"],
```
Comment (vegaro) [inline fastlane/Fastfile:8]: Do you need to add `\` before the `'`? I don't think you do so you can probably just do

```suggestion
    'RevenueCatPurchasesCapacitor.podspec' => ["'PurchasesHybridCommon', '{x}'"],
```

Comment (tonidero) [reply fastlane/Fastfile:8]: You're right! Good catch!

File: fastlane/Fastfile [hunk 2]
```diff
@@ -193,7 +193,7 @@ end
 
 def get_phc_version
   Dir.chdir(get_root_folder) do
-    return (sh "cat RevenueCatPurchasesCapacitor.podspec | grep \'\"PurchasesHybridCommon\", \' | awk \'{print($3)}\' | sed \"s/\'//g\"").strip
+    return (sh "cat RevenueCatPurchasesCapacitor.podspec | grep \"\'PurchasesHybridCommon\', \'\" | awk \'{print($3)}\' | sed \"s/\'//g\"").strip
```
Comment (vegaro) [inline fastlane/Fastfile:196]: same here, I don't think `'` needs the `\`

```suggestion
    return (sh "cat RevenueCatPurchasesCapacitor.podspec | grep \"'PurchasesHybridCommon', '\" | awk '{print($3)}' | sed \"s/\'//g\"").strip
```

**Review by vegaro — APPROVED** [review]

File: fastlane/Fastfile (hunk 2 above)
Comment (tonidero) [reply fastlane/Fastfile:196]: Yup, just tested to confirm and it works fine 👍

**Pull request merged**